guides/v2.1/cloud/cdn/cloud-vcl-custom-snippets.md

```diff
@@ -11,223 +11,304 @@ functional_areas:
+  | `dynamic` | Indicates if this is a [regular snippet](https://docs.fastly.com/vcl/vcl-snippets/using-dynamic-vcl-snippets/) which is included in the versioned VCL for the Fastly service configuration, or a [dynamic snippet](https://docs.fastly.com/vcl/vcl-snippets/using-dynamic-vcl-snippets/) which can be modified and deployed without requiring a new VCL version.
+  | `active`  | Indicates if the snippet or version is activated and in use. Returns `true` or `false`. Make note of the version number for an active snippet. Use this to clone the version. 
+  {:style="table-layout:auto;"}
+  
 
-## Clone the active VCL version and all snippets {#clone}
 
-Clone the version using the active version number. This creates a copy of all existing VCL snippets for that version using a new version number. After you clone the version, you can [modify and add](#create-snippet) VCL snippets. Save the new version number for the bash script.
+## Manage custom VCL snippets from the Admin UI
+
+You can [add custom VCL snippets](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/CUSTOM-VCL-SNIPPETS.md) from the *Fastly Configuration* > *Custom VCL Snippets* section in the Admin UI.
+
+![Manage custom VCL snippets]
+
+The *Custom VCL snippets* view shows only the snippets added through the Admin UI. If snippets are added using the Fastly API, [manage them using the API](#manage-vcl).
+
+See the following examples that show how to create and manage custom VCL snippets using the Admin UI:
+
+- [Set up redirects to WordPress using Fastly]({{ page.baseurl }}/cloud/cdn/fastly-vcl-wordpress.html)
+- [Block referral spam]({{ page.baseurl }}/cloud/cdn/fastly-vcl-badreferer.html)
+
+
+## Manage custom VCL snippets using the API
+
+The following walk-through shows you how to create regular VCL snippet files and add them to your Fastly service configuration using the Fastly API. You can create and manage the snippets from the _terminal_ application. You do not need an SSH connection into a specific environment. 
+
+**Prerequisites**
+
+-  Configure your {{ site.var.data.ece }} environment for Fastly services. See [Set up Fastly]({{ page.baseurl }}/cloud/cdn/configure-fastly.html). 
+
+-  [Get Fastly API credentials]({{ page.baseurl }}/cloud/cdn/configure-fastly.html) to authenticate requests to the Fastly API. Make sure that you get the credentials for the correct environment, Staging or Production.
+
+-  Save Fastly service credentials as bash environment variables that you can use in cURL commands:
+ 
+   ```
+   export FASTLY_SERVICE_ID=<Service ID>
+   ```
+   ```
+   export FASTLY_API_TOKEN=<API Token>
+   ```
+
+   The exported environment variables are available only in the current bash session and are lost when you close the terminal. You can redefine variables by exporting a new value. To view the list of exported variables related to Fastly:
```

Comment:
Yes-- I thought about this,  but since the examples use the variable it seems worth including. Also, in my experience documenting APIs, including this information helped people get fewer errors on their API operations.